marcelo_zoccoler/entry_user_interf3/Readme.md

```diff
@@ -255,26 +263,33 @@ viewer.window.add_dock_widget(flood_widget, area='right')         # Add our gui
 
 <img alt="figure 3" id="figure3" src="images/napari_flood_tool3.png" />
 
-Neat! Just a couple of things missing though. We only get Spinboxes now, what about the Slider? Don't worry, we can fix this still maintaining the one-liner, although a bigger one now 😬 . We can add widget options as python dictionaries, like this:
+Neat! Just a couple of things missing though. We only get Spinboxes now, what about the Slider? Don't worry, we can fix this still maintaining the "one-liner", although a bigger one now 😬 . We can add widget options as python dictionaries, like this:
 
 ```Python
 flood_widget = magicgui(flood, delta={'label': 'Temperature Increase (Δ°C):', 
                                            'min': 0, 'max' : 3, 'step': 0.1},
                                 new_level={'label':'Sea Level (dm):', 'widget_type':'Slider',
-                                           'min': 0, 'max' : 255})
+                                           'min': 0, 'max' : 255, 'enabled' : False},
+                                auto_call=True)
 ```
 
-Now, you get practically the same result as the earlier approach. If you don't know the widget options, take a look [here](https://napari.org/magicgui/usage/widget_overview.html) to find the widget you want and discover its parameters.
+Now, you get practically the same result as the earlier approach. We even discarded the button by using `auto_call=True`. If you don't know the widget options, take a look [here](https://napari.org/magicgui/usage/widget_overview.html) to find the widget you want and discover more about its parameters.
```

Comment:
So if I get it correctly, `auto_call=True` will automatically call the associated functions whenever a widgets is changed/clicked/etc? Maybe you could point this out a bit more clearly - if the associated function is not very fast, using `auto_call` could lead to trouble and probably Napari freezing until all queued calls to the function behind the plugin are processed?

Reply:
Good catch! I updated the text to warn the reader about that possibility.